### ISSUE TITLE
URGENT. not possible to get pointnet++ part-seg results.

### ISSUE THREAD

**Issue by a18700** [yanx27/Pointnet_Pointnet2_pytorch#52]
Hello, is there any one got pointnet++ part-seg results, as noted in README.md?

I only get nearby ~=79 / ~=84 for avg-class miou / avg-ins miou.

avg-ins miou is acceptable, i don't know why avg-class miou is deterior.

**Comment by yanx27** [comment]
Hi @a18700, thank you for your attention. Did you download right dataset of alignment version? Or right torch version? I test several times that can achieve at least 85 in part seg. There are also some people can reproduce this result see [issue #28](https://github.com/yanx27/Pointnet_Pointnet2_pytorch/issues/28) :)

**Comment by a18700** [comment]
> Hi @a18700, thank you for your attention. Did you download right dataset of alignment version? Or right torch version? I test several times that can achieve at least 85 in part seg. There are also some people can reproduce this result see [issue #28](https://github.com/yanx27/Pointnet_Pointnet2_pytorch/issues/28) :)

Hello,

I got a good results as you noted,

however, having troubles in transporting DGCNN to this baseline code.

Does this codes in shapenet- part segmetation includes normalization of inputs? it might hurt a information about absolute position.

**Comment by yanx27** [comment]
@a18700 Yes, this code use alignment part seg data and normalize each sample to a unit ball. This setting is same with official PointNet++ paper. Besides, during past experiment, this will achieve better results comparing to using  absolute position :)

**Comment by a18700** [comment]
> @a18700 Yes, this code use alignment part seg data and normalize each sample to a unit ball. This setting is same with official PointNet++ paper. Besides, during past experiment, this will achieve better results comparing to using absolute position :)

Hello, about another problem..

When loading model and optimizer in checkpoint, we found out loaded models starts from quite deteriorated condition.

For 100 eps, model got 80/84 but after retrieving, drop to 73/83

This is strange.. can you guess reasons behind?

**Comment by yanx27** [comment]
@a18700 You can load model directly by setting the same `--log_dir`,  if you load model by yourself, it maybe drop accuracy because different learning rate (I set learning rate decay during training).

**Comment by a18700** [comment]
> @a18700 You can load model directly by setting the same `--log_dir`, if you load model by yourself, it maybe drop accuracy because different learning rate (I set learning rate decay during training).

hello, really thank you quick reply.

However, i think learning rate is not an issue because
1. I also tried to load optimizer.state.dict / and also learning rate changing codes are inside iteration codes of each batch.
2. And also tested using test_partseg.py, get same deteriorated results.

Any other reason possible for this phenomena or is really a learning rate issue?

**Comment by yanx27** [comment]
@a18700 == I did not face with this situation, did you retrain model with right dataset and retry model loading?
